{
    "repository": "Farama-Foundation/Minari",
    "stars": 119,
    "language": "python",
    "size": 1935,
    "clone_url": "https://github.com/Farama-Foundation/Minari.git",
    "timestamp": "2023-06-28T18:30:16.016688Z",
    "clone_success": true,
    "number_of_actions": 6,
    "number_of_test_actions": 1,
    "actions_successful": false,
    "actions_build_tools": [
        "unknown",
        "unknown",
        "unknown",
        "unknown",
        "pytest",
        "unknown"
    ],
    "actions_test_build_tools": [
        "pytest"
    ],
    "actions_run": {
        "failed": true,
        "tests": [],
        "stdout": "[2bee289f-2e61-4497-bfcc-a575828e8f1c/build] \ud83d\ude80  Start image=crawlergpt:latest\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build]   \ud83d\udc33  docker pull image=crawlergpt:latest platform= username= forcePull=false\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build]   \ud83d\udc33  docker create image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build]   \ud83d\udc33  docker run image=crawlergpt:latest platform= entrypoint=[\"tail\" \"-f\" \"/dev/null\"] cmd=[]\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build]   \ud83d\udc33  docker exec cmd=[chown -R 1012:1013 /tmp/ad9c0f7e-1596-11ee-8a50-bb14de238602/Farama-Foundation-Minari] user=0 workdir=\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build] \ud83e\uddea  Matrix: map[python-version:3.7]\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build] \u2b50 Run Main actions/checkout@v3\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build]   \u2705  Success - Main actions/checkout@v3\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build] \u2b50 Run Main docker build -f bin/Dockerfile \\\n  --build-arg PYTHON_VERSION=3.7 \\\n  --tag minari-docker .\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build]   \ud83d\udc33  docker exec cmd=[bash --noprofile --norc -e -o pipefail /var/run/act/workflow/1] user= workdir=\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build]   | Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post \"http://%2Fvar%2Frun%2Fdocker.sock/v1.24/build?buildargs=%7B%22PYTHON_VERSION%22%3A%223.7%22%7D&cachefrom=%5B%5D&cgroupparent=&cpuperiod=0&cpuquota=0&cpusetcpus=&cpusetmems=&cpushares=0&dockerfile=bin%2FDockerfile&labels=%7B%7D&memory=0&memswap=0&networkmode=default&rm=1&shmsize=0&t=minari-docker&target=&ulimits=null&version=1\": dial unix /var/run/docker.sock: connect: permission denied\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build]   \u274c  Failure - Main docker build -f bin/Dockerfile \\\n  --build-arg PYTHON_VERSION=3.7 \\\n  --tag minari-docker .\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build] exitcode '1': failure\n[2bee289f-2e61-4497-bfcc-a575828e8f1c/build] \ud83c\udfc1  Job failed\n",
        "stderr": "Error: Job 'build' failed\n",
        "workflow": "/tmp/ad9c0f7e-1596-11ee-8a50-bb14de238602/Farama-Foundation-Minari/.github/workflows/build-crawler.yml",
        "build_tool": "pytest",
        "elapsed_time": 269.70907640457153
    }
}